src/test/java/net/snowflake/ingest/streaming/internal/BlobBuilderTest.java

```diff
@@ -23,18 +27,21 @@ public void testSerializationErrors() throws Exception {
     BlobBuilder.constructBlobAndMetadata(
         "a.bdec",
         Collections.singletonList(createChannelDataPerTable(1, false)),
-        Constants.BdecVersion.THREE);
+        Constants.BdecVersion.THREE,
+        true);
```

Comment:
also test with false, make the testclass into a parameterized class?

Reply:
Make to parameterized class.